mlir/lib/Dialect/Linalg/IR/LinalgOps.cpp

```diff
@@ -3176,8 +3175,8 @@ LogicalResult WinogradOutputTransformOp::verify() {
       return emitOpError("expect input width equals to input tile size");
     expectedOutputShape[2] = (rightTransform ? m : 1) * valueTileW;
   }
-  expectedOutputShape[0] = valueShape[4];
-  expectedOutputShape[3] = valueShape[5];
+  expectedOutputShape[0] = valueShape[getValueNDim()];
+  expectedOutputShape[3] = valueShape[getValueFDim()];
```

Comment:
nit: use the extraClassDeclarations for the expectedOutputShape too in the verifier.

Reply:
Done.